### PR TITLE
Suggest a license to upstream authors

### PR DESCRIPTION
**Pull request by pulkitsinghal**
I would like to suggest that you add a license to this nodejs module so its usage isn't dubious.

### CODE REVIEW

**Comment by eduohe** [comment]
Hi @vedmalex nice work! Please can you merge the @pulkitsinghal PR?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Pulkit Singhal
```
[the remark this answers is not in the document]
Comment (vedmalex) [reply LICENSE]: ок

[the remark this answers is not in the document]
Comment (vedmalex) [reply LICENSE]: library with version bump and license updated is in npm

[the remark this answers is not in the document]
Comment (eduohe) [reply LICENSE]: Thanks @vedmalex and @pulkitsinghal

**Pull request merged**